modules/admin_manual/pages/configuration/server/excluded_blacklisted_files.adoc

```diff
@@ -21,6 +20,18 @@ file explorer, the path cannot be accessed.
 For example configurations please see
 `owncloud/config/config.sample.php`.
 
+[NOTE]
+====
+Many filesystems are, by default, case-sensitive. 
+This means that multiple files or folders with the same name cannot exist in the same directory.
+Therefore no differentiation is made in processing, as it would just return a deny at a later stage.
+With this behaviour you get an immediate error message.
+
+Example: The storage backend has a reserved directory name ".snapshot" which is excluded.
+If you try to add a file via the browser or sync a file from the client named ".snapshot",
+you get an immediate ownCloud triggered deny.
```

Comment:
Where you wrote "an immediate ownCloud triggered deny", do you mean that ownCloud blocks the file upload?

Reply:
yes, compared when owncloud just handles back the deny from the backend storage system

Reply:
So ownCloud returns a response indicating that the upload/update failed?

Reply:
See my explanation above